environments/se/core/src/main/java/org/jboss/weld/environment/se/WeldContainer.java

```diff
@@ -123,6 +123,23 @@ public static WeldContainer instance(String id) {
     }
 
     /**
+     * A convenient method for {@link CDI.current()}. Returns current {@link WeldContainer} instance
+     * if there is exactly one instance running. Throws {@link IllegalStateException} in any other case.
+     * @return Current {@link WeldContainer} instance if and only if exactly one instance exists
+     * @throws {@link IllegalStateException} if there is 0 or more than 1 instance running
+     */
+    public static WeldContainer current() {
+        List<String> ids = WeldContainer.getRunningContainerIds();
```

Comment:
I know, I used that as I think it's a safe way to handle lists.
But you are right that here we can simply use the original list; I will change the code.